cabal.project

```diff
@@ -54,166 +56,206 @@ package typed-protocols-cbor
 package lobemo-backend-monitoring
   tests: False
 
+package network-mux
+  tests: False
+
+package ouroboros-consensus
+  tests: False
+
+package ouroboros-network
+  tests: False
+
+package typed-protocols
+  tests: False
+
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-crypto
   tag: 4590efa638397e952a51a8994b5543e4ea3c1ecd
+  --sha256: 0hl2n3bba5v2j0lmxhs7hs01z3aznh2bwf9cb434icq2g0bl8ms3
```

Comment:
Firstly I thing that should have a better name that `sha256`. I would suggest `nix-sha256` to make it obvious what it is.

Secondly, (and this is *really* important) if I do not use Nix locally, how do I update these hashes?

Reply:
```
$ ssh user@some-nix-host nix-prefetch-git --quiet https://github.com/input-output-hk/cardano-crypto 4590efa638397e952a51a8994b5543e4ea3c1ecd |jq -r .sha256
0hl2n3bba5v2j0lmxhs7hs01z3aznh2bwf9cb434icq2g0bl8ms3
```

Maybe we can automate this with a regexp over the `cabal.project`?